src/index.css

```diff
@@ -1,7 +1,276 @@
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+:root {
+  --pale-gray: #f5f6fa;
+  --white-dark: #fafafa;
+  --white-light: #fff;
+  --body-back: rgb(237, 234, 234);
+  --blue-azure: #0290ff;
+  --gray-dark: #888;
+  --black-dark: #121212;
+}
+
+h1 {
+  font-size: x-large;
+}
+
 body {
   font-family: 'Montserrat', sans-serif;
+  background-color: var(--pale-gray);
+  height: 100%;
+}
+
+.container {
+  width: 90%;
+  margin: auto;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.3);
+}
+
+header,
+.list .books li {
+  background-color: var(--white-light);
+}
+
+header {
+  margin: 5%  auto 0 auto;
+  width: 100%;
+  padding: 20px 0;
+}
+
+nav {
+  width: 85%;
+  margin: auto;
+  justify-content: space-between;
+}
+
+nav h1 {
+  color: var(--blue-azure);
+}
+
+nav,
+nav ul {
+  align-items: center;
+}
+
+nav,
+nav ul,
+.list .books .book-card,
+.list .books li .book-card .button-card,
+.form-add-book,
+.card-layout,
+.percentage {
+  display: flex;
+}
+
+nav ul {
+  gap: 20px;
+}
+
+nav ul li a {
+  color: var(--gray-dark);
+}
+
+nav ul li .active {
+  color: var(--black-dark);
+  font-weight: 500;
+}
+
+nav button {
+  border-radius: 50%;
+  padding: 10px 12px;
+  border: none;
+}
+
+nav button i {
+  color: var(--blue-azure);
+}
+
+main {
+  width: 100%;
+  padding: 20px 0;
+  margin: 0 auto 4% auto;
+  background-color: var(--body-back);
+}
+
+.chapter .chapter-card small {
+  font-size: 11px;
+  text-transform: uppercase;
+  color: var(--gray-dark);
+}
+
+.chapter button {
+  gap: 20px;
+}
+
+.list .books {
+  width: 85%;
+  margin: auto;
+}
+
+.list .books li {
+  width: 100%;
+  margin: 13px 0;
+  padding: 10px 0;
+}
+
+.list .books li .book-card {
+  flex-direction: column;
+  width: 50%;
+  margin: auto;
+}
+
+.list .books li .book-card p {
+  font-size: large;
+  font-weight: 500;
+}
+
+.list .books li .book-card small {
+  color: var(--blue-azure);
+  padding: 0 0 15px 0;
+}
+
+.list .books li .book-card .button-card {
+  padding: 10px 0;
+  gap: 20px;
+}
+
+.list .books li .book-card .button-card button {
+  background-color: transparent;
+  color: var(--blue-azure);
+  font-size: medium;
+  font-family: 'Roboto Slab', serif;
+  font-weight: 400;
+}
+
+.list .books li .book-card .button-card button:hover {
+  color: #0562a9;
+}
+
+.list .books li .book-card .button-card button:not(:last-child) {
+  border-top: none;
+  border-left: none;
+  border-bottom: none;
+  border-color: rgb(246, 245, 245);
+  padding: 0 10px 0 0;
+}
+
+.list .books li .book-card .button-card button:last-child {
+  border: none;
+  padding: 0 10px;
+}
+
+.line {
+  width: 85%;
+  background-color: #fbf2f2;
+  height: 2px;
+  border: none;
+  margin: auto;
+}
+
+.newbook {
+  width: 85%;
+  margin: 20px auto;
+}
+
+.newbook h1 {
+  color: var(--gray-dark);
+  font-weight: 500;
+  padding: 10px 0;
+}
```

Comment:
- [ ] The colour of the horizontal line labelled `11` is different from the one in your design, please correct it as it is important for developers to closely follow designs.
- [ ] Kindly correct `12` to `Category` as shown in the design

![Screenshot 2023-06-21 at 16 08 48 (2)](https://github.com/zbahati/BookStore-React/assets/64774936/d9995219-8378-4815-b6fe-9bd07ff1c263)